Wikipedia/Code/WMFArticleViewController.m

```diff
@@ -705,6 +710,7 @@ - (void)fetchArticleForce:(BOOL)force {
 
 - (void)fetchArticle {
     [self fetchArticleForce:YES];
+    [_pullToRefresh endRefreshing];
```

Comment:
When the refresh is ended the progress bar takes over and hence there is no need for showing the  activity indicator.